plugins/modules/ecs_service.py

```diff
@@ -608,6 +621,8 @@ def update_service(self, service_name, cluster_name, task_definition,
         # desired count is not required if scheduling strategy is daemon
         if desired_count is not None:
             params['desiredCount'] = desired_count
+        if enable_execute_command:
+            params['enableExecuteCommand'] = enable_execute_command
```

Comment:
I'm not familiar with ECS.  By defaulting to False I suspect it won't be possible to disable this option once it's set...

 ```suggestion
        if enable_execute_command is not None:
            params['enableExecuteCommand'] = enable_execute_command
```